.github/workflows/pypi-release.yaml

```diff
@@ -76,7 +76,7 @@ jobs:
         run: |
           sleep 3
           python -m pip install --upgrade pip
-          python -m pip install --index-url https://test.pypi.org/simple --upgrade xarray
+          python -m pip install --extra-index-url https://test.pypi.org/simple --upgrade xarray
```

Comment:
It appears this will likely pull a previous version instead of [the most recently published](https://github.com/pydata/xarray/runs/2519982391?check_suite_focus=true)... How should we address this?

Reply:
if that doesn't help, we can also install manually and check that it works